allensdk/brain_observatory/behavior/behavior_project_cache/tables/sessions_table.py

```diff
@@ -123,6 +123,10 @@ def postprocess_additional(self):
                 self._df['date_of_acquisition_behavior']
             self._df = self._df.drop(['date_of_acquisition_behavior',
                                       'date_of_acquisition_ophys'], axis=1)
+            # Enforce an integer type on ophys_session_id due to many
+            # columns being NaN.
+            self._df['ophys_session_id'] = self._df['ophys_session_id'].astype(
```

Comment:
Not sure I understand the comment here. Isn't it returned as int from the database?

Reply:
I'll clarify. This ends up being a pandas issue as you are merging in a column that doesn't have values for all the behavior_sessions.

Reply:
Did not know about the Int64 dtype. Thanks